docs/assets/sass/docs.scss

```diff
@@ -1,4 +1,4 @@
-@import "node_modules/govuk-frontend/all";
+@import "../../../app/assets/sass/extensions";
```

Comment:
Do we need to try and load extensions in docs? I think it would only need Frontend?